src/main/java/com/openlattice/authorization/PermissionsApi.java

```diff
@@ -38,20 +36,30 @@
     String CONTROLLER = "/permissions";
     String BASE       = SERVICE + CONTROLLER;
 
-    String EXPLAIN    = "/explain";
+    String EXPLAIN = "/explain";
+    String UPDATE = "/update";
 
     /**
      * Add, removes, or sets the ace for a particular acl key. Successful only if user is the owner of acl key.
-     * 
+     *
      * @param req The acl key, the principals, and the aces to set for that particular ace key.
      * @return The aces for the acl key, after applying the request changes.
      */
     @PATCH( BASE )
     Void updateAcl( @Body AclData req );
 
+    /**
+     * Add, removes, or sets the ace for a particular set of acl keys. Successful only if user is the owner of all acl keys.
+     *
+     * @param req The acl key, the principals, and the aces to set for that particular ace key.
+     * @return The aces for the acl key, after applying the request changes.
```

Comment:
The return type is Void, also for simple `updateAcl`

Reply:
yesss you're right, thanks!